extensions/amp-ooyala-player/0.1/test/test-amp-ooyala.js

```diff
@@ -70,69 +71,143 @@ describes.realWin(
       });
     }
 
-    it('renders a V3 player', () => {
-      return getOoyalaElement(
-        'Vxc2k0MDE6Y_C7J5podo3UDxlFxGaZrQ',
-        '6440813504804d76ba35c8c787a4b33c',
-        '5zb2wxOlZcNCe_HVT3a6cawW298X'
-      ).then((player) => {
-        const playerIframe = player.querySelector('iframe');
-        expect(playerIframe).to.not.be.null;
-        expect(playerIframe.src).to.equal(
-          'https://player.ooyala.com/iframe.html' +
-            '?platform=html5-priority&ec=Vxc2k0MDE6Y_C7J5podo3UDxlFxGaZrQ' +
-            '&pbid=6440813504804d76ba35c8c787a4b33c'
+    describe('rendering', async () => {
+      it('renders a V3 player', () => {
+        return getOoyalaElement(
+          'Vxc2k0MDE6Y_C7J5podo3UDxlFxGaZrQ',
+          '6440813504804d76ba35c8c787a4b33c',
+          '5zb2wxOlZcNCe_HVT3a6cawW298X'
+        ).then((player) => {
+          const playerIframe = player.querySelector('iframe');
+          expect(playerIframe).to.not.be.null;
+          expect(playerIframe.src).to.equal(
+            'https://player.ooyala.com/iframe.html' +
+              '?platform=html5-priority&ec=Vxc2k0MDE6Y_C7J5podo3UDxlFxGaZrQ' +
+              '&pbid=6440813504804d76ba35c8c787a4b33c'
+          );
+        });
```

Comment:
I'd recommend changing this test & below tests like the following to make it easier to read:
```
const videoId = ...;
return getOoyalaElement(videoId)
  .then((player) => {
    ...
    expect(playerIframe).not.to.be.null;
    expect(playerIframe.src).to.contain(videoId);
  });
```